### PR TITLE
calculating the file count for a collection by querying mediaflux

### LINKED ISSUE #391: Query and display file count on project contents page
Screenshot 2024-01-24 at 11 32 37 AM

- This is already calculated by Mediaflux, and stored within a metadata field for each Collection

Acceptance Criteria:
- [ ] Retrieve the count of files associated with the Collection from the Mediaflux server API
- [ ] Propagate this through the View

### PR DESCRIPTION
**Pull request by JaymeeH**
closes #391

### CODE REVIEW

**Comment by JaymeeH** [comment]
Instead of calculating the total number of files and collection every time a page is visited -- which will take an extreme amount of time with an extreme amount of files -- use an accumulator to store the value and query that.

**Comment by JaymeeH** [comment]
Running into an rspec error for disabled http requests? 
/spec/system/project_show_spec.rb:54

File: spec/rails_helper.rb [hunk 1]
```diff
@@ -12,6 +12,7 @@
 require "webmock/rspec"
 WebMock.disable_net_connect!(allow_localhost: true,
                              allow: "chromedriver.storage.googleapis.com")
+# WebMock.enable_net_connect!
```
Comment (hectorcorrea) [inline spec/rails_helper.rb:15]: ???

**Review by hectorcorrea — APPROVED** [review]

**Comment by hectorcorrea** [comment]
@JaymeeH Looks good. I made a suggestion and left a comment but it feel free to merge.

File: spec/rails_helper.rb (hunk 1 above)
Comment (JaymeeH) [reply spec/rails_helper.rb:15]: This was to allow the testing of http requests in the project show rspec

**Pull request merged**